doc/api/policy.md

```diff
@@ -111,8 +111,8 @@ To generate integrity strings, a script such as
 `node -e 'process.stdout.write("sha256-");process.stdin.pipe(crypto.createHash("sha256").setEncoding("base64")).pipe(process.stdout)' < FILE`
 can be used.
 
-Integrity can be specified as the boolean value `true` to accept any
-body for the resource which can be useful for local development. It is not
+Integrity can be specified as the boolean value `true` to accept anybody
```

Comment:
This change is incorrect. "Anybody" means "any person", but here we're talking about response bodies not human bodies. It should stay as "any body".

Reply:
Thank you so much